lib/utils/CountryHelper.dart

```diff
@@ -1003,4 +1003,19 @@ extension OpenFoodFactsCoutryExtension on OpenFoodFactsCountry {
   };
 
   String get iso2Code => _ISO_2_CODES[this]!;
+
+  bool containsCountryCode(String countryCode) =>
+      _ISO_2_CODES.containsValue(countryCode);
+
+  OpenFoodFactsCountry? fromCountryCode(String countryCode) {
+    OpenFoodFactsCountry? country;
+
+    _ISO_2_CODES.forEach((key, value) {
+      if (value == countryCode) {
+        country = key;
+        return;
```

Comment:
The `return` here is misleading I think.

Reply:
It just stops the for loop from running further than it has to, adding a comment

Reply:
I don't think it does stop the loop. The next loop will be for the next (key, value). Add a print, just for tests, in order to double-check that.

Reply:
Thats interesting it actually doesn't work, (It works in normal for loops)

Also while testing I found out that the extension doesn't work on the enum itself 

So not `OpenFoodFactsCountry.fromCountryCode`
but `OpenFoodFactsCountry.FRANCE.fromCountryCode` works.

Reply:
The `return` exits from the anonymous function, that is called for all (key, value)s.
And the method should be `static`.

Reply:
Just created a new CountryHelper class and put it in there